src/lib/DpApi.js

```diff
@@ -78,29 +85,28 @@ const dpRpc = function (method, parameters, id = null) {
     jsonrpc: '2.0',
     method: method,
     id: id === null ? uuid() : id,
-    params: parameters
+    params: parameters,
   }
 
   return doRequest({
-    method: 'post',
     url: Routing.generate('rpc_generic_post'),
-    data
+    method: 'POST',
+    data,
+    params: {
+      headers: {
+        'Content-type': 'application/json'
+      }
+    }
   })
 }
 
 /**
  * Perform an external API call without any default headers
  */
-const externalApi = function (url) {
-  const contentType = axios.defaults.headers.common['Content-Type']
-  delete axios.defaults.headers.common['Content-Type']
+const externalApi = async (url, data) => {
+  const response = await fetch(url, data)
```

Comment:
Shouldn't we wrap this in a try/catch block to be able to handle an error, should it occur?

Reply:
I don't know why I added the `.json`. I removed it again. With that change imho we don't need a try catch here anymore, since in case of an error the method will return it and it can be handled in the calling method.

Reply:
a1b124f472872fd5f170944b77f98418ae0d5f6b